README.md

```diff
@@ -8,13 +8,13 @@ network.
 
 Requirements:
 * Crossplane should be installed.
-* [GCP Stack](https://github.com/crossplane/stack-gcp) should be installed and its version should be at least 0.6.0
+* [GCP Provider](https://github.com/crossplane/stack-provider) should be installed and its version should be at least 0.6.0
```

Comment:
Typo here. Should say `provider-gcp`

Reply:
also maybe bump to v0.7.0 here and definitely v0.7.1 for AWS when we get to that stack.